docs/src/examples/components/Chat/Types/ChatMessageExampleStyled.shorthand.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react'
 import { Avatar, Chat, Provider } from '@stardust-ui/react'
 
+const reactions = [{ icon: 'thumbs up', count: 5 }, { icon: 'thumbs down', count: '1K' }]
```

Comment:
nit-nit: just for the sake of keep optimistic and opened to `Hi` messages, lets rebalance numbers of `thumbs up` and `thumbs down` reactions ;)

Reply:
Will do